packages/wp-story-editor/src/api/story.js

```diff
@@ -73,7 +73,7 @@ const transformGetStoryResponse = (post) => {
   post.taxonomies = links?.['wp:term']?.map(({ taxonomy }) => taxonomy) || [];
   post.terms = embedded?.['wp:term'] || [];
```

Comment:
General note:

Seems like we're still returning things like `post._embedded` and `post._links` here, even though we shouldn't 🤔

Reply:
Went through all the API callback responses which go to the core editor, and found out that only `_links` is being used for these callbacks, will delete them.

```js
getAuthors: [
  {
    _links: {},
  }
]

updateCurrentUser: {
  _links: {}
}

getTaxonomyTerm: [
  {
    _links: {}
  }
]
```